packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/SwapOrderTx/model.tsx

```diff
@@ -17,10 +17,15 @@ export const getDestination = (props: Props) => {
   switch (props.order.kind.direction) {
     case 'TO_USERKEY':
     case 'ON_CHAIN':
-      return getOutput(props.order) + ' Wallet'
+      return (
+        props.supportedCoins[getOutput(props.order)].displayName + ' Wallet'
+      )
     case 'FROM_USERKEY':
     case 'INTERNAL':
-      return getOutput(props.order) + ' Trading Wallet'
+      return (
+        props.supportedCoins[getOutput(props.order)].displayName +
```

Comment:
Maybe try pulling `props.supportedCoins[getOutput(props.order)].displayName` into a variable and use template string interpolation instead of `+`